code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -64,6 +64,7 @@
 	var/obj/item/default_storage = /obj/item/weapon/storage/backpack/dufflebag/drone //If this exists, it will spawn in internal storage
 	var/obj/item/default_hatmask //If this exists, it will spawn in the hat/mask slot if it can fit
 	var/seeStatic = 1 //Whether we see static instead of mobs
+	var/seeAlerts = 1
```

Comment:
@Firecage nothing, in the end it compiles down to 1 and 0

The important thing is that if we see TRUE/FALSE we can more easily make the assumption that the variable is a boolean on/off flag of some description

It also makes the code nicer to read.